### PR TITLE
Implement books page.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#113

### CODE REVIEW

File: haskala/modules/custom/haskala_book/plugins/content_types/books_list/books_list.tpl.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="letter-katalog">
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/books_list/books_list.tpl.php]: Please change letter-katalog to letters, Also in jekyll

**Comment by HelenaEksler** [comment]
PR is ready for the code review.

Test is failing because of the PhantomCSS.

video3

**Comment by bitamar** [comment]
@HelenaEksler - Please update the phantomcss images of the elements we changed (In the drupal tests). If you don't know how, probably @DavidGizra can help

**Comment by HelenaEksler** [comment]
@bitamar Tests are green. So after review we can merge it.

**Comment by bitamar** [comment]
@HelenaEksler - Please re-create the books filter screenshot (With the updated design)

**Comment by HelenaEksler** [comment]
Chrome:
selection_186

Firefox:
selection_185

**Comment by HelenaEksler** [comment]
@bitamar 
Is it OK like this?
selection_194

**Comment by HelenaEksler** [comment]
Merged.

**Pull request merged**